worker/src/RTC/Transport.cpp

```diff
@@ -608,6 +608,35 @@ namespace RTC
 				break;
 			}
 
+			case Channel::Request::MethodId::TRANSPORT_SET_MAX_OUTGOING_BITRATE:
+			{
+				auto jsonBitrateIt = request->data.find("bitrate");
+
+				// clang-format off
+				if (
+					jsonBitrateIt == request->data.end() ||
+					!Utils::Json::IsPositiveInteger(*jsonBitrateIt)
+				)
+				// clang-format on
+				{
+					MS_THROW_TYPE_ERROR("missing bitrate");
+				}
+
+				this->maxOutgoingBitrate = jsonBitrateIt->get<uint32_t>();
+
+				MS_DEBUG_TAG(bwe, "maximum outgoing bitrate set to %" PRIu32, this->maxOutgoingBitrate);
+
+				request->Accept();
+
+				if (this->tccClient)
+				{
+					this->tccClient->SetMaxOutgoingBitrate(this->maxOutgoingBitrate);
+					ComputeOutgoingDesiredBitrate();
+				}
```

Comment:
This is not ok. `tccClient->SetMaxOutgoingBitrate()` may throw if given value is less than `initialAvailableOutgoingBitrate`, so we must properly reject the request with error if that happens.

```suggestion
				if (this->tccClient)
				{
					uint32_t bitrate = jsonBitrateIt->get<uint32_t>();

					// NOTE: This may throw if given bitrate is less than current
					// initialAvailableOutgoingBitrate, so don't update things before
					// calling this method.
					this->tccClient->SetMaxOutgoingBitrate(bitrate);
					this->maxOutgoingBitrate = bitrate;

					MS_DEBUG_TAG(bwe, "maximum outgoing bitrate set to %" PRIu32, this->maxOutgoingBitrate);

					ComputeOutgoingDesiredBitrate();
				}

				request->Accept();
```